packages/loader/container-loader/src/container.ts

```diff
@@ -1559,11 +1560,8 @@ export class Container
 		);
 
 		// If we saved ops, we will replay them and don't need DeltaManager to fetch them
-		const sequenceNumber =
+		const lastProcessedSequenceNumber =
 			pendingLocalState?.savedOps[pendingLocalState.savedOps.length - 1]?.sequenceNumber;
```

Comment:
We want the delta manager to skip fetching any ops we already have and have processed, but we don't want it to move its `initialSequenceNumber` forward as well. Otherwise, what will happen is that the refresh of the virtualized datastore snapshots will be thought of as the past as their sequence number will still point to the base sequence snapshot.

For example:
1. We are at snapshot seq 10 with a virtualized snapshot at seq 50
3. We refresh up to seq 100. Our snapshot and virtualized snapshot are both at 100.
4. We generate 2 online ops that we have seen round tripped
5. We generate 1 offline op
6. We stash these changes
7. We apply these stashed changes to a new container
8. The container will load its delta manager with initial sequence number at 102.
9. When loading the virtualized snapshot offline, it will be loaded at sequence number 100.
10. The runtime will think that it downloaded a snapshot from the past when in fact it is as recent as it can get.

When we reload this data